pkgs/tools/misc/abduco/default.nix

```diff
@@ -1,14 +1,20 @@
-{ lib, stdenv, fetchFromGitHub, writeText, conf ? null }:
+{ lib, stdenv, fetchpatch, fetchzip, writeText, conf ? null }:
 
-stdenv.mkDerivation rec {
-  pname = "abduco";
+let
+  name = "abduco";
+  owner = "martanne";
+  rev = "8c32909a159aaa9484c82b71f05b7a73321eb491";
+in
+stdenv.mkDerivation {
+  pname = name;
```

Comment:
Assigning name creates confusion as pname+version gets combined into name, so pname is usually not equal to name.